gui/additionsPane.py

```diff
@@ -79,9 +81,12 @@ def __init__(self, parent):
         self.gangPage = GangView(self.notebook)
         self.notebook.AddPage(self.gangPage, "Fleet", tabImage = gangImg, showClose = False)
 
+        self.notes = NotesView(self.notebook)
+        self.notebook.AddPage(self.notes, "Notes", tabImage = notesImg, showClose = False)
+
         self.notebook.SetSelection(0)
 
-    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet"]
+    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet", "Notes"]
```

Comment:
I've noticed one small bug and I can't figure out how to fix it.

If your preferences are set to reload previously open fits, when this opens the fit, it goes to the "Fleet" tab. I'm really not sure why or what code is setting that.